### DISCUSSION TITLE
Documentation for Slicing

### DISCUSSION THREAD

**Discussion by tanzatanza** [vlang/v#7356] [General]
In Python trying to get a slice that doesn't exist always(?) gives an empty list:
```
>>> a = [12]
>>> a[1]
IndexError: list index out of range
>>> a[1:]
[]
>>> a[2:]
[]
```
V's behavior is a little different, but the documentation doesn't mention this?! Will `a[1..]` still not panic in the future here? Or will `a[n..]` perhaps not panic in the future, either?
```
>>> a := [12]
>>> a[1]
V panic: array.get: index out of range (i == 1, a.len == 1)
>>> a[1..]
[]
>>> a[2..]
V panic: array.slice: invalid slice index (2 > 1)
```

**Comment by medvednikov** [comment]
It's always going to panic, like in Go:

```
panic: runtime error: slice bounds out of range [5:1]

goroutine 1 [running]:
main.main()
	/tmp/sandbox097229313/prog.go:9 +0x48
```

I think Python has this behavior because of negative indexing.

**Reply by tanzatanza** [reply]
Okay, thank you.

**Comment by medvednikov** [comment]
This should be documented.
